rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/210_knn_search_profile.yml

```diff
@@ -0,0 +1,104 @@
+setup:
+  - requires:
+      cluster_features: "mapper.vectors.bbq"
+      reason: 'kNN float to better-binary quantization is required'
```

Comment:
```  
- requires:
      reason: 'Quantized vector rescoring is required'
      test_runner_features: [capabilities]
      capabilities:
        - method: GET
          path: /_search
          capabilities: [knn_quantized_vector_rescore]
```

Pretty sure this should require the capability, not bbq

Reply:
🤦 yes, BwC tests showed me that. I corrected it and added some more testing when using similarity in a8633e523ae7e1bbce110247023f45c658cafb20